contracts/APM.sol

```diff
@@ -298,19 +277,42 @@ contract APM is IAPM, GovernanceOwnable {
                 path[i],
                 path[i + 1]
             );
-            amounts[i + 1] = getAmountOut(amounts[i], reserveIn, reserveOut);
+            amounts[i + 1] = _getAmountOut(amounts[i], reserveIn, reserveOut);
         }
     }
 
-    // Bank Access
-    function removeLiquidity(
+    // force reserves to match balances
+    function sync(address tokenAddress) public {
+        totalReserve[tokenAddress] = IERC20(tokenAddress).balanceOf(address(this));
+    }
+
+
+    function _removeLiquidity(
         address _to,
         address tokenAddress,
         uint256 amount
-    ) external onlyBank {
+    ) private {
         // transfer
         IERC20(tokenAddress).safeTransfer(_to, amount);
         // update getReserves
         updateWhenRemoveLiquidity(amount, tokenAddress);
     }
+
+    // Bank Access
+    function removeLiquidityBank(
```

Comment:
u dont need to create 2 functions with same implementation (removeLiquidityBank, removeLiquidityGovernance) with different access... just modify the access with one function